src/main/java/lotto/WinRanks.java

```diff
@@ -0,0 +1,44 @@
+package lotto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class WinRanks {
+    Map<Integer, Rank> winMap;
+
+    public WinRanks() {
+        winMap = new HashMap<>();
+        winMap.put(3, new Rank(3, 5000, 0));
+        winMap.put(4, new Rank(4, 50000, 0));
+        winMap.put(5, new Rank(5, 1500000, 0));
+        winMap.put(6, new Rank(6, 2000000000, 0));
```

Comment:
```suggestion
        winMap.put(6, new Rank(6, 2_000_000_000, 0));
```

언더바로 구분을 해주면 가독성이 상승합니다~

Reply:
넵!!